Update integrate-payone.md (#1420)
spryker/spryker-docs: docs/pbc/all/payment-service-providers/payone/integrate-payone.md

If the app was connected successfully, a corresponding message appears, and the app status changes to **Connected**. The payment methods you've selected in step 8, appear in **Administration&nbsp;<span aria-label="and then">></span>  Payment methods**:
![payone-credit-card](https://spryker.s3.eu-central-1.amazonaws.com/docs/aop/user/apps/payone/payone-credit-card.png).

> ## 3. Whitelist Payone Domain

Reviewer: We use allowlist/denylist instead of whitelist/blacklist. Also, these can be used as nouns only. I rephrased the title accordingly.
```suggestion
## 3. Add Payone domain to your allowlist
```

Author: The problem is that here we reference a technical term inside the code; see the code example below. I am fully on to call it allowlist instead of whitelist but please double-check the understandability of the doc with both elements being displayed.

Reviewer 2: I think as long as the technical example is clear, there is no problem with calling it allowlist/denylist as the meaning of both are the same.

> To enable Payone to redirect your customers to their 3D Secure page and later one to your success page, you must add Payone domain inside your **Content Security Policy**. To do that please change `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.

Reviewer 2: ```suggestion
To enable Payone to redirect your customers to their 3D Secure page and later one to your success page, you must add ACP domain inside your **Content Security Policy** whitelist. To do that please change `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```

The page isn't owned by PayOne, but is owned by us. I suppose this does not matter too much for end users.

> To enable Payone to redirect your customers to their 3D Secure page and later one to your success page, you must add ACP domain inside your **Content Security Policy** whitelist. To do that please change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.

Reviewer: ```suggestion
To enable Payone to redirect your customers to their 3D Secure page and later to your success page, you must add the ACP domain inside your **Content Security Policy** allowlist. To do that, change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
```

> Changes required in the `deploy.yml` file.

Reviewer: ```suggestion
 In the `deploy.yml` file, introduce the required changes:
```

```yml
image:
  environment:
    SPRYKER_AOP_APPLICATION: '{
      "APP_DOMAINS": [
        "os.apps.aop.spryker.com",
        ...
      ],
      ...
    }'
```

> Changes required in the `config/Shared/config_default.php` file.

Reviewer 2: ```suggestion
Alternatively, you may whitelist the domain from the `config/Shared/config_default.php` file. If you updated the `deploy.yml` file, then this step can be ignored.
```

> Alternatively, you may whitelist the domain from the `config/Shared/config_default.php` file. If you updated the `deploy.yml` file, then this step can be ignored.

Reviewer: ```suggestion
Alternatively, you may add to the allowlist the domain from the `config/Shared/config_default.php` file. If you updated the `deploy.yml` file, this step can be ignored.
```
